RestaurantReservation.Db/Models/Positions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace RestaurantReservation.Db.Models;
+
+public enum Positions
+{
+    Employee,
```

Comment:
NIT: Could you update the enum to include an explicit integer value for each member? This ensures clarity and avoids potential issues if the enum order changes in the future.